.github/workflows/CI.yml

```diff
@@ -99,6 +99,8 @@ jobs:
     runs-on: ubuntu-22.04
     container:
       image: "ubuntu:22.04"
+    env:
+      ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
```

Comment:
Couldn't we use secrets in the code directly? If not, is this safe? I think it should be just didn't know if this is a normal thing to do.

Reply:
I tried `Environment.GetEnvironmentVariable "secrets.GITHUB_TOKEN"` and `Environment.GetEnvironmentVariable "GITHUB_TOKEN"` but they didn't work. I didn't find a way to do this. Do you know a way to use secrets in the code directly without defining it as an environment variable?

Reply:
No, I don't know any way for this. But thinking more about this I think the way you've implemented this should be ok. Because I'm not sure if accessing the secrets from a script is logical anyway. :+1:

Reply:
where do we stand here?

Reply:
@knocte Do you think defining an environment variable is the right way to use GITHUB_TOKEN in the script?

Reply:
Yes

Reply:
Why don't you share?

Reply:
We can pass the access token as an input argument to the script but I don't think it's a better way than this one

Reply:
let's leave it like this then, so where do we stand with this PR? is it tested?

Reply:
Yes it's tested

Reply:
And when testing this, you didn't need to add any secret right? it will just work out of the box?

Reply:
Right. It should work